types/office-js-preview/index.d.ts

```diff
@@ -37988,6 +38639,15 @@ declare namespace Excel {
         set(properties: Interfaces.CommentUpdateData, options?: OfficeExtension.UpdateOptions): void;
         /** Sets multiple properties on the object at the same time, based on an existing loaded object. */
         set(properties: Excel.Comment): void;
+        /**
+         * Assigns the task attached to the comment to the given user as the sole assignee. If there is no task, one will be created.
+         *
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         *
+         * @param email The email address of the user to assign the task.
```

Comment:
here and other places
```suggestion
         * @param email The email address of the user to assign the task to.
```